lib/options.py

```diff
@@ -297,6 +297,21 @@ def __init__(self):
                 last position), test-run will show a list of tags
                 and stop.
                 """)
+        parser.add_argument(
+                '--env',
+                dest='show_env',
+                action='store_true',
+                default=False,
+                help="""Print environment variables, which are set by test-run.
+
+                Useful for, say, running built-in luatest executable.
+
+                Usage: just source it into a current environment:
+
+                ```
```

Comment:
Three backticks will look bad because it will be one line. Please use one backtick.

Reply:
Ok, let's proceed in https://github.com/tarantool/test-run/pull/337.